python/chronos/dev/test/run-installation-options.sh

```diff
@@ -25,18 +25,23 @@ if [ -z "${OMP_NUM_THREADS}" ]; then
     export OMP_NUM_THREADS=1
 fi
 
-ray stop -f
-
-MOPTIONS=$1
-
-echo "Running chronos tests TF1 and Deprecated API"
-echo $MOPTIONS
-python -m pytest -v -m "${MOPTIONS}" test/bigdl/chronos/forecaster
+# ray stop -f
+
+if [ $1 = true ]; then
+    echo "Running chronos[pytorch] tests"
+    python -m pytest -v -m "not tf2 and not onnxrt16 and not automl and not distributed and not diff_set_all" test/bigdl/chronos/data \
```

Comment:
we have decided to have 6 ops as we discussed offline
- op_torch
- op_tf2
- op_automl
- op_distributed
- op_inference
- op_diff_set_all

Reply:
I agree we use the above 6 ops, but I think we can do this in later pr (test [tensorflow],[distributed],[inference] options and add these ops based on this pr).